content/en/blog/_posts/2019-06-24-kubeadm-ha-v115.md

```diff
@@ -2,13 +2,11 @@
 layout: blog
 title: 'Automated High Availability in kubeadm v1.15: Batteries Included But Swappable'
 date: 2019-06-24
+author: >
+  Lucas Käldström ([@luxas](https://github.com/luxas) SIG Cluster Lifecycle co-chair & kubeadm subproject owner, Weaveworks),
+  Fabrizio Pandini ([@fabriziopandini](https://github.com/fabriziopandini) kubeadm subproject owner, Independent)
```

Comment:
Always write “independent” in lower case, unless they represent a company named something like **Independent, Inc.**

Reply:
```suggestion
  Lucas Käldström ([@luxas](https://github.com/luxas) (Weaveworks),
  Fabrizio Pandini ([@fabriziopandini](https://github.com/fabriziopandini) (independent)
```